.github/workflows/update-changelog.yaml

```diff
@@ -0,0 +1,57 @@
+name: '[Release] Update CHANGELOG.md'
+on:
+  release:
+    types: [released]
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  setup:
+    # this workflow will always fail in forks; bail if this isn't running in the upstream
+    if: github.repository == 'awslabs/amazon-eks-ami'  
+    runs-on: ubuntu-latest
+    outputs:
+      tag_name: ${{ steps.variables.outputs.tag_name }}
+    steps:
+    - id: variables
+      run: |
+        echo "tag_name=$(echo ${{ github.ref }} | cut -d/ -f3)" >> $GITHUB_OUTPUT
+  update-changelog:
+    runs-on: ubuntu-latest
+    needs:
+      - setup    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: awslabs/amazon-eks-ami
+          ref: refs/heads/master
+          path: amazon-eks-ami/
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('fs');
+            const changelogPath = './amazon-eks-ami/CHANGELOG.md';
```

Comment:
We should verify that the changelog doesn't already have the release before we add it to ensure that this workflow is idempotent. If it fails, we'd ideally open a github issue.

Reply:
I added a check for the release title. We'll get an email if this workflow fails, I *could* open an issue if you feel strongly about it

Reply:
Not really. I think that the changelog has limited value, if any, since it's already in the release notes and now it's just a formatted list of commits and version information.